test/src/loaders/workspace_uploader/workspace_uploader_test.py

```diff
@@ -203,7 +212,7 @@ def test_post_process(setup_and_teardown):
 
     host_assembly_dir = params.assembly_dirs[0]
     assembly_name = ASSEMBLY_NAMES[0]
-    src_file = params.target_files[0]
```

Comment:
is this cruft? Seems like it's not used

Reply:
it's not used since this test only test assembly. But used in genome cases in the separate test.

Reply:
:+1: